tests/unit/Endpoints/Webhooks/ProductHandlerTest.php

```diff
@@ -142,7 +268,8 @@ public function testOrderOnlySyncMode()
             self::UPDATE_DATADUMP_HOOK,
             'ShopModule::ShopProduct',
             'events',
-            false
+            false,
+            StoreKeeperOptions::SYNC_MODE_ORDER_ONLY
```

Comment:
Can you add a test for the stock change to this test?

Reply:
Changed test and added testing of other details for products.